itests/src/test/resources/testconfiguration.properties

```diff
@@ -7,6 +7,7 @@ minimr.query.files=\
 # Queries ran by both MiniLlapLocal and MiniTez
 minitez.query.files.shared=\
   compressed_skip_header_footer_aggr.q,\
+  create_database.q,\
```

Comment:
let's use TestMiniLlapLocalCliDriver if the patch has nothing specific to tez container mode

Reply:
Got it. Good to know :)

Reply:
Removed the Tez tests and squashed the commits.